### PR TITLE
I18n redesign

### PR DESCRIPTION
**Pull request by yagotome**
This PR redesigns project for i18n by:

- Using gatsby-plugin-i18n (as mentioned in #3)
- Creating a page for each language that statically fetches content from CMS for that language and pass data through the _page component (such as the old single page file)_ 
- Creating a switcher component boilerplate for _non-localized_ route that render page localized according to user's browser's language (e.g. if user accesses `lutick.com/about` instead of `lutick.com/pt/about` or `lutick.com/en/about`, but their browser's language is for example `pt`, it renders same page to `lutick.com/pt/about` although the route is still `lutick.com/about`)
- Setting up i18next selected language for each language route
- Creating a context for current language
- Creating a `LocalizedLink` that uses the language from the react's context in order to redirect to correct language URL

### CODE REVIEW

File: src/data/languages.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  langs: ['en', 'pt'],
```
Comment (BelfDev) [inline src/data/languages.js]: Neat!

File: src/lib/i18n.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ i18n
     interpolation: {
       escapeValue: false,
     },
+
+    react: {
+      useSuspense: false,
```
Comment (BelfDev) [inline src/lib/i18n.js]: What's this?

File: src/pages/index.pt.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import IndexPage from '../components/IndexPage';
+import LanguageContext from '../contexts/LanguageContext';
+
+const IndexPage_pt = () => {
```
Comment (BelfDev) [inline src/pages/index.pt.js]: Nice, ultimately I think we reached a nice compromise.

File: src/lib/i18n.js (hunk 2 above)
Comment (yagotome) [reply src/lib/i18n.js]: This indicates to react-i18next lib that we are not using [React Suspense](https://reactjs.org/docs/react-api.html#suspense) and so react-i18next does not crash trying to trigger Suspense.

**Pull request merged**